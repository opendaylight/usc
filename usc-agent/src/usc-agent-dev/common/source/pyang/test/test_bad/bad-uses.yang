module bad-uses {
  namespace "urn:test:cyclic-groupings";
  prefix "cyclic-groupings";
  
  grouping target {
    leaf address {
      uses target; // error: 'uses' not allowed in 'leaf'
      type int8;
    }
  }
}

