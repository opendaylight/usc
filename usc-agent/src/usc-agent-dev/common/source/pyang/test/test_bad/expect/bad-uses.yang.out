bad-uses.yang:7: error: UNEXPECTED_KEYWORD
